### PR TITLE
[I18n][14.0] mail_thread:  fix dialog create res.partner is not translated

### PR DESCRIPTION
**Pull request by daonamutc1**
Before this commit:
When the user clicks on the checkbox in the mail thread, a dialog will be shown, the content in this dialog is not translated.
When opening the dialog, the system has not passed the context to dialog.
After this commit:
The content will be translated.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by seb-odoo** [comment]
Hello, I'm not able to reproduce the issue, the dialog is correctly translated in a v14 runbot even without your fix.

**Comment by seb-odoo** [comment]
Ok nevermind, it's only partially translated, I was looking at the title.

I guess this is fine then.

File: addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 odoo.define('mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js', function (require) {
 'use strict';
 
+var session = require('web.session');
```
Comment (seb-odoo) [inline addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js:4]: use `const`

File: addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js [hunk 2]
```diff
@@ -128,7 +130,9 @@ class ComposerSuggestedRecipient extends Component {
                 this._dialogRef.comp.widget.on('closed', this, () => {
                     this._isDialogOpen = false;
                 });
-                this._dialogRef.comp.widget.open();
+                var widget = this._dialogRef.comp.widget;
+                widget.context = Object.assign({}, widget.context, session.user_context)
```
Comment (seb-odoo) [inline addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js:134]: It's strange to have to do this here specifically, I wonder if it shouldn't be part of `FormViewDialogComponentAdapter` or genrally `ComponentAdapter`, but I guess it's fine here as a small fix.

**Review by seb-odoo — CHANGES_REQUESTED** [review]

File: addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js (hunk 2 above)
Comment (daonamutc1) [reply addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js:134]: Here we add to the widget some user context. User context may or may not be used by other components. If not used it is just a few small variables.

File: addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 odoo.define('mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js', function (require) {
 'use strict';
 
+const session = require('web.session');
```
[the remark this answers is not in the document]
Comment (daonamutc1) [reply addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js]: @seb-odoo I moved it.

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**